### ISSUE TITLE
The form doesn't work because all of the check-boxes are disabled

### ISSUE THREAD

**Issue by abgordon** [cla-assistant/cla-assistant#532]
URL to the linked Repo/Org:
https://cla-assistant.io/Comcast/Ravel?pullRequest=4

Steps to reproduce the problem:

Open form

What is the expected result?
I can sign the form via the check boxes

What happens instead?

disabled attribute shown; can't click anything 

<img width="983" alt="Screen Shot 2020-02-11 at 11 26 32 AM" src="https://user-images.githubusercontent.com/3271877/74266468-95921780-4cc1-11ea-8ea5-8eb4930f48a4.png">

**Comment by tjrgg** [comment]
I can't reproduce this issue.

**Comment by KharitonOff** [comment]
Unfortunately I can not reproduce the issue too... @abgordon do you still face it?

**Comment by abgordon** [comment]
@KharitonOff I do. What do you recommend?

**Comment by KharitonOff** [comment]
I assume that you've already signed the cla and now the fields are disabled because you can accept the cla only once.

The status of the PR https://github.com/Comcast/Ravel/pull/4 remains "pending" because github can not recognize you as a committer. If you add the email address, which you have used for the commits to your github account then your commits will be associated with your github account and cla status should also be updated

**Comment by abgordon** [comment]
Revisiting this, I added `agordo240@COSML-1686904.local` which appears to be something my managed laptop added automatically to my local git. Because it's not an actual email address,  I can't verify it, so i'm not sure if adding it to my account will help the status move out of pending.

**Comment by KharitonOff** [comment]
Hmmm... usually you don't need to verify the email address in order to assign these commits to your account. I don't know why it is not working for you now, the email address seems to be the right one.
I see two ways to solve this: 
- convince the project owner to accept your PR despite of technical issue with the email address, you have lastly accepted the CLA
- follow the instructions here in order to reset your git settings and adjust commit history: https://help.github.com/en/github/using-git/changing-author-info

**Comment by abgordon** [comment]
Update of current state. So it seems that we need to modify the commit 

<img width="674" alt="Screen Shot 2020-03-02 at 3 36 51 PM" src="https://user-images.githubusercontent.com/3271877/75724460-acfa6a00-5c9b-11ea-9592-a3d057ffc302.png">

**Comment by KharitonOff** [comment]
Hi Andrew, I'm very sorry about it! I should have warned you that the second option causes reset of the git history, my fault! I hope you managed to restore it. 
I am very confused, that even this step didn't help to assign your commits to your github account.

Unfortunately we can not do much about it on our side, since we fully on information, which GitHub provides us. This means if GitHub doesn't assign these commits to your account, cla assistant can't recognize you as a committer. The gray octocat icon near commits signals such cases as described here: https://help.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Comment by KharitonOff** [comment]
Here is the result of reproduced graphQL call, which cla assistant does in order to collect committers of your PR:
![Screen Shot 2020-03-03 at 15 19 40](https://user-images.githubusercontent.com/8480657/75784550-a551c880-5d62-11ea-9a09-0da48292c036.png)

**Comment by abgordon** [comment]
Fixed it. The tutorial recommends doing this with a clean repo, which given the state of the repo was a terrible idea. 

I freshly-cloned a repo which meant the `.git/refs` directory was not fully populated. When I rewrote history and force-pushed, it did not rewrite the history of any of the outstanding branches, basically making them orphaned - PRs opened in github from the PR branches showed the error "no changes to show, these two branches have entirely different histories."

Given the outstanding 1000+ lines of code in PR, I panicked a bit and re-pushed the state from the original repo. This meant the author history was not rewritten, but got the repo back to its original state, minus the PRs, but they could be re-opened at least.

This morning I rewrote the history from my working repo with the refs directory populated, and then force pushed that. Fixed the CLA state but still lost the comments & PR's. 

So all good here, lost some comments on the working PR but no code is gone. In the future I would make a note of that for people instead of just directing them to the tutorial. **If there are outstanding working branches and pull requests, doing the history rewrite from a bare repo will result in code loss.** I would recommend doing a full copy with `cp -r <repo> <repo>-save` and attempting to rewrite history with a backup on your machine.

My 2 cents, but obviously, I'm not as good at git as I thought I was if I did what I did yesterday. Not sure the rationale of starting from a bare repo in the tutorial.
